src/main/java/org/folio/innreach/repository/InnReachTransactionRepository.java

```diff
@@ -33,4 +34,7 @@
 
   @Query(name = FETCH_OPEN_BY_LOAN_ID_QUERY_NAME)
   Optional<InnReachTransaction> fetchOneByLoanId(UUID folioLoanId);
+
+  @Query(name = FETCH_BY_LOAN_ID_QUERY_NAME)
+  Optional<InnReachTransaction> fetchTransactionByLoanId(UUID folioLoanId);
```

Comment:
why `fetchOneByLoanId` is not used instead?

Reply:
It's obvious that they are different, the question is why different query is applied here. 
Pay attention to the requirements:
> mod-inn-reach should check if the loan is associated with **_an active_** INN-Reach transaction

Reply:
This query uses for testing or better not to use such query ?

Reply:
Resolved